datascience/tables.py

```diff
@@ -86,7 +86,19 @@ def from_rows(cls, rows, labels):
 
     @classmethod
     def from_records(cls, records):
-        """Create a table from a sequence of records (dicts with fixed keys)."""
+        """Create a table from a sequence of records (dicts with fixed keys).
+
+           Args:
+
+               records: A list of dictionaries with same keys.
+
+           Returns:
+
+               If the list is empty, it will return an empty table.
+               Otherwise, it will return a table with the labels as the column name, and the corresponding data.
```

Comment:
Replace "labels" with "the dictionary's keys" (labels and columns are synonyms in this library)

Reply:
Done. Thank you for your response.